src/styles/main.scss

```diff
@@ -1,3 +1,244 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+/*#region header*/
 body {
   margin: 0;
 }
+
+header {
+  width: 100%;
+  height: 60px;
+  display: flex;
+  justify-content: space-between;
+  box-sizing: border-box;
+  align-items: center;
+  padding-right: 49px;
+  border: 1px solid #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  margin-left: 49px;
+}
+
+.nav {
+  padding: 0;
+  margin: 0;
+  height: 60px;
+  display: flex;
+  align-items: center;
+
+  &__list {
+    margin: 0;
+    padding: 0;
+    display: flex;
+  }
+
+  &__item {
+    display: flex;
+    list-style: none;
+    text-decoration: none;
+    height: 60px;
+  }
+
+  &__link {
+    height: 60px;
+    text-transform: uppercase;
+    font-family: "Roboto", sans-serif;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 14px;
+    color: #060b35;
+    text-decoration: none;
+    display: flex;
+    margin: 0 0 0 20px;
+    position: relative;
+    align-items: center;
+
+    &:hover {
+      color: rgba(0, 172, 220, 1);
+
+    }
+  }
+}
+
+.is-active {
+  color: rgba(0, 172, 220, 1);
+  margin: 0;
+
+  &::after {
+    position: absolute;
+    content: "";
+    display: block;
+    width: 100%;
+    top: 56px;
+    left: 0;
+    bottom: 0;
+    background: rgba(0, 172, 220, 1);
+    border-radius: 8px;
+    height: 4px;
+  }
+}
+/*#endregion header*/
+
+/*#region card*/
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+:root {
+  --blue: #00acdc;
+  --dark: #060b35;
+}
+
+html {
+  font-family: "Roboto", sans-serif ;
+}
+
+body,
+h2,
+p {
+  margin: 0;
+}
+
+.card {
+  padding: 32px 16px 16px;
+  width: 200px;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  border: 1px solid #f3f3f3;
+  box-sizing: border-box;
+  margin: 0 48px 46px 0;
```

Comment:
Don't add external styles (positioning or margins) to BEM-blocks. Use mix where necessary and move all external styles under element selector.